app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
```

Comment:
This is just a style point, but I generally try to use the single-line, trailing `if` in scenarios like this: `do_stuff if time_is_right?`.